src/workerd/api/kv.h

```diff
@@ -13,9 +13,12 @@ class KvNamespace: public jsg::Object {
   // A capability to a KV namespace.
 
 public:
-  explicit KvNamespace(uint subrequestChannel): subrequestChannel(subrequestChannel) {}
+  explicit KvNamespace(uint subrequestChannel,
+      kj::Array<kj::Tuple<kj::String, kj::String>> additionalHeaders = {})
```

Comment:
super nit: Kinda weird to swap the swap the parameter and field order for no reason, and we try to stay away from tuples (in favor of structs with named fields), but I don't know how someone could confuse a tuple of headers... so :shrug:

Reply:
Yeah please don't use `kj::Tuple` unless you're doing variadic template metaprogramming. A struct with two members is much more readable.

Reply:
Got it. The nit reason I swapped the field order is that putting a uint first introduces padding & I can't have a first parameter be default. I can put additional headers first if I get rid of it. Let me know.

Reply:
Went with option 2 - rearrange it so that the array comes first & updated `workerd-api` to reflect that.